kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,12 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    def __init__(self, app):
+        DjangoWhiteNoise.__init__(self, app)
+
+    # Correct some MIME types
+    EXTRA_MIMETYPES = (
+            ('application/font-woff', '.woff'),
+            ('application/font-woff2', '.woff2'),)
```

Comment:
Ah ok, I didn't see that they were also setting this and that this value was so sparse. I imagined it having a lot more. This seems fine but perhaps with a caveat if whitenoise adds more in the future?